lib/yt/audit.rb

```diff
@@ -29,5 +29,15 @@ def self.has_subscribe_annotations?(video_id)
         annotation.link && annotation.link[:type] == :subscribe
       end
     end
+
+    # Audit youtube association of a video
+    # @param [String] video_id the video to audit.
+    # @return [Boolean] if the video description has link to its own channel.
+    def self.has_youtube_association?(video_id)
```

Comment:
Is this the name that is used in the report? Can we come up with a better name?

In my opinion, "has YouTube association" is too generic to understand what this method does.